frappe/tests/test_translate.py

```diff
@@ -45,7 +45,9 @@ def tearDown(self):
 
 	def test_extract_message_from_file(self):
 		data = frappe.translate.get_messages_from_file(translation_string_file)
-		exp_filename = "apps/frappe/frappe/tests/translation_test_file.txt"
+		bench_path = get_bench_path()
+		file_path = frappe.get_app_path("frappe", "tests/translation_test_file.txt")
```

Comment:
```suggestion
		file_path = frappe.get_app_path("frappe", "tests", "translation_test_file.txt")
```

I guess we won't run on windows any time soon, but why presume to know the path separator if we can avoid it...

Reply:
Incorporated in [force-push](https://github.com/frappe/frappe/compare/9d075afbb92a9b5916d28f00c2dc0145391a90c6..fb11f9a8fa275a0b6793e75f86871a4f6bef7bb1)